helm/chart/epinio/README.md

```diff
@@ -0,0 +1,104 @@
+# Epinio Helm Chart
+
+From app to URL in one command.
+
+## Introduction
+
+This helm chart can be used to deploy Epinio on a cluster. It is an alternative
```

Comment:
```suggestion
This helm chart deploys Epinio on a cluster. It is an alternative
```

The `epinio install` command is gone, is it not ?So this here is a replacement, not an alternative.